doc/crm.8.adoc

```diff
@@ -2094,7 +2091,14 @@ Usage:
 delete <node>
 ...............
 
+.Deprecation note
+*****
+This command is deprecated in favor of `crm cluster remove [-F] -c <node>`,
+which will also stop cluster services and adjust related pacemaker configs.
+*****
```

Comment:
@zzhou1 Could you please take a look at this man doc change?

Reply:
How about this:
"""
This command is deprecated and in favor of `crm cluster remove [-F] -c <node>`,
which will adjust the related cluster configurations and clean up the leftover
(eg. stopping the cluster services) on the removed node.
"""